src/librbd/Journal.cc

```diff
@@ -50,7 +50,8 @@ struct C_DecodeTag : public Context {
 
   int process(int r) {
     if (r < 0) {
-      lderr(cct) << "failed to allocate tag: " << cpp_strerror(r) << dendl;
+      lderr(cct) << this << " " << __func__ << " "
```

Comment:
Usually we have a colon after func name (`__func__ << ": "`), while you have only space in most of your modifications (there are still cases when you left a colon). Is it intentional?